pages/common/opt.md

```diff
@@ -0,0 +1,15 @@
+# opt
+
+> LLVM .bc -> .bc modular optimizer and analysis printer. `opt` takes LLVM source files as input, runs the specified optimizations or analyses on it, and then outputs the optimized file or the analysis results.
+
+- Run an optimization or an analysis in a bitcode file:
+
+`opt {{-passname}} {{file.bc}} -S -o {{file_opt.bc}}`
+
+- Print the Control Flow Graph of a function to a "dot" file:
+
+`opt {{-dot-cfg}} -S {{file.bc}} -disable-output`
+
+- Optimize the program with "-O2" and write its output to another file:
```

Comment:
Perhaps we could specify what `-O2` actually does instead? How about this?

```suggestion
- Optimize the program at level 2 and output the result to another file:
```

Also, perhaps `file_opt.bc` can be `path/to/file_opt.bc` instead?